filament/src/Skybox.cpp

```diff
@@ -47,6 +47,8 @@ struct Skybox::BuilderDetails {
     SkyboxType mType = Skybox::SkyboxType::ENVIRONMENT;
     float mUiScale = 1.0f;
     UpDirectionAxis mUpDirectionAxis = UpDirectionAxis::Y_UP;
+    float2 mCheckerboardGrays{1.0, 0.8};
+    float4 mGradientSettings{0.8, 0.93, 0.55, 0.4};
```

Comment:
Relatively nit, but I'd add the f postfix after the float literals.

Reply:
Fixed in https://github.com/shapr3d/filament/pull/14/commits/c26d1a69c98809f22d4eb15d018d8c8c45b79777